### PR TITLE
Add lockfile

### PR DESCRIPTION
**Pull request by Irving-Betterment**
**Summary of changes:**
Commit lockfile(s) previously ignored (see https://bundler.io/guides/faq.html#using-gemfiles-inside-gems, https://github.com/thoughtbot/appraisal/issues/68#issuecomment-28083719)

/task https://app.asana.com/0/0/1206244989881250/f

### CODE REVIEW

File: jekyll-watch.gemspec [hunk 1]
```diff
@@ -20,8 +20,7 @@ Gem::Specification.new do |spec|
     spec.add_runtime_dependency "wdm", "~> 0.1.0"
   end
 
-
-  spec.add_development_dependency "bundler", "~> 1.6"
```
Comment (Irving-Betterment) [inline jekyll-watch.gemspec:24]: I unpinned this old version, as current version manages gems ok.

**Comment by effron** [comment]
Do we use this anywhere? We might be able to remove this repo if this is unused

**Comment by Irving-Betterment** [comment]
@effron [an org-wide search](https://github.com/search?q=org%3ABetterment+jekyll-watch&type=code) says it's not, as the few dependants pull the gem from rubygems.org ([ie](https://github.com/Betterment/engineering-blog/blob/main/Gemfile.lock#L24)).

The same applies to https://github.com/Betterment/jekyll-autoprefixer. We can safely remove or set them read-only.

**Comment by Irving-Betterment** [comment]
I will close this PR while we discuss with other stakeholders if we can safely decommission them.

**Pull request closed without merging**